src/main/java/bsise/server/report/daily/dto/DailyPublishedStaticsDto.java

```diff
@@ -0,0 +1,7 @@
+package bsise.server.report.daily.dto;
+
+public interface DailyPublishedStaticsDto {
```

Comment:
```suggestion
public interface DailyReportStatisticsDto {
```

클래스 이름이 오타인 것 같습니다
데일리 리포트 통계치를 의미하는 것 같아 이렇게 수정했습니다